conbench/app/_plots.py

```diff
@@ -584,7 +584,7 @@ def time_series_plot(
         toolbar_location="right",
         x_range=(t_start, t_end),
     )
-    p.toolbar.logo = None
+    p.toolbar.logo = None  # type: ignore[attr-defined]
```

Comment:
I don't know why mypy started complaining about this. 

`error: "Instance[Toolbar]" has no attribute "logo"  [attr-defined]`

Reply:
missing/insufficient stubs, yeah -- we'll have to endure treating false negatives.

Reply:
> So my question was more "why did these errors just start showing up today?" I didn't think I touched anything relevant to these errors.

Ah, makes sense! Thanks for clarifying.

> As a precaution, I'm going to update our CI so it installs the webapp dependencies too for linting.

Yes, please! Maybe it's worth separating the `mypy` check out then. `ci/lint` was / can be so fast because of very little dependencies an _fast_ tooling.  So far, it completed within ~30 s, and this can be even faster.

The type checking can by definition never be so fast when it (of course) requires all libs installed, and has sometimes even some computational work to do. We can do that separately.

Reply:
@austin3dickey it's always impressive how quickly one forgets... :) In #627 I did think of this problem space, and that thinking led to the naming of `requirements-dev.txt`, and I added type check dependencies there: https://github.com/conbench/conbench/pull/627/commits/e9a126298c206ac0d4fc4e33d7f1366268ac9627

I suppose the answer is we want CI to have the same environment that we also use locally in our IDEs, but on the other hand we don't want to (need to) have those `types-*` packages in our prod environment. 

We can try to resolve all this at a later point in time.